### ISSUE TITLE
Feature request: "Use website icon" option in host settings

### ISSUE THREAD

**Issue by MurzNN** [ethanliu/location-switcher-addon#15]
Will be good to add option "Use website icon" in host settings, for show original icon of host, instead of custom. And also skip replacing icon to custom in "Advanced" version of extension.

**Comment by ethanliu** [comment]
How about just use the website's favicon url as custom image url, if both are the same icon, the tab icon should looks the same no matter "Replace favicon" is enable or not. 
Is that make sense to you?

<img width="552" alt="image" src="https://user-images.githubusercontent.com/65704/134643084-6424d2e3-a86b-4fd3-99ff-cffd64a50e6a.png">

**Comment by MurzNN** [comment]
Yes, I already use that way, but searching the favicon url and fill it manually for each website is annoying process :)

**Comment by ethanliu** [comment]
another workaround would be using google favicon snatcher api, but it won't work with local domains.
i.e. https://www.google.com/s2/favicons?sz=32&domain=www.google.com

instead of dynamic parsing the favicon url from current tab every time, fetch the favicon url and save as custom image url from preference is better.  however it might need other permissions for getting external webpage contents from preference.

**Comment by MurzNN** [comment]
Yeah, great idea, also there is a similar service from yandex: https://favicon.yandex.net/favicon/v2/github.com

**Comment by ethanliu** [comment]
I add two buttons beside the custom button, it does nothing but generate the custom image url with the specified domain.

![demo](https://user-images.githubusercontent.com/65704/135439982-6a4acecd-42a0-44f1-b9ba-58a3a6766342.gif)
